hack/conformance-test.sh

```diff
@@ -98,13 +96,19 @@ echo "Conformance test checking conformance with Kubernetes version 1.0"
 #  Restart: node management.
 #  Scale: performance
 #  Services.*load balancer: not all cloud providers have a load balancer.
-#  Services.*NodePort: flaky
+#  Services.*NodePort: requires you to open the firewall yourself, so not covered.
+#  Services.*nodeport: requires you to open the firewall yourself, so not covered.
 #  Shell: replies on optional ssh access to nodes.
 #  SSH: optional feature.
+#  Addon\supdate: requires SSH
 #  Volumes: contained only skipped tests.
-export CONFORMANCE_TEST_SKIP_REGEX="Cadvisor|MasterCerts|Density|Cluster\slevel\slogging.*|Etcd\sfailure.*|Load\sCapacity|Monitoring|Namespaces.*seconds|Pod\sdisks|Reboot|Restart|Nodes|Scale|Services.*load\sbalancer|Services.*NodePort|Shell|SSH|Volumes"
+#  Clean\sup\spods\son\snode: performance
+#  MaxPods\slimit\snumber\sof\spods: not sure why this wasn't working on GCE but it wasn't.
+#  Kubectl\sclient\sSimple\spod: not sure why this wasn't working on GCE but it wasn't
```

Comment:
Well, actually, it all passes but the following:

Kubectl client Simple pod [It] should support port-forward

So maybe we can just filter out that one test?